existing-machines/standard.json

```diff
@@ -65,10 +65,10 @@
         "ip_address": "10.2.1.115",
         "node_type": "hadoop-cm"
     },
-    "bastion": {
-        "ip_address": "10.2.1.116",
-        "node_type": "bastion",
-        "public_ip_address": "10.2.0.101",
+    "gateway": {
```

Comment:
I know they're only example IPs but why skip from 115 to 117

Reply:
in fact I note that mgr2 and mgr3 have the same IP also, so fix that too if you don't mind?